plugin/Renderer/InputFieldCheckboxRenderer.php

```diff
@@ -110,7 +113,7 @@ private function buildFieldsCollection(): FieldsCollection
 	public function render()
 	{
 		$pFieldsCollection = $this->buildFieldsCollection();
-
+		$hintText = !empty($this->getHint()) ? '<p class="hint-text">' . $this->getHint() . '</p>' : "";
```

Comment:
I wish there was a different way to do this with escaping by default. It would be no problem if the link wasn't there, right? Can you think of any safe options?

Reply:
One option would be to set the link using Javascript. Yayy or nayy?

Reply:
Using Javascript will be hard for doing multiple languages.

Reply:
Wouldn't be the first time `%s` is replaced within Javascript:
https://github.com/onOfficeGmbH/oo-wp-plugin/blob/f48788277992582364e6b41967f43dc71d73813a/js/admin.js#L134

Reply:
this issue, logic same in function render() in file plugin/Renderer/InputFieldCheckboxRenderer.php and i update it